README.md

```diff
@@ -133,11 +138,21 @@ some meaningful syntax in the source file. This option is enabled by default.
 
 ## Developer's Guide
 
-To set up and build from the command line, perform the following steps:
-1. Install dependencies using `yarn install`.
-2. Run `yarn global add vsce`.
-3. In the root of the project, run `vsce package` to generate a `.vsix` file 
-4. Run `code --install-extension my-extension-0.0.1.vsix`, replacing `my-extension-0.0.1.vsix` with the created `.vsix` file. 
+To set up your development environment, perform the following steps:
+1. Install the newest Node.js version for your operating system (OS); see the [Node.Js homepage](https://nodejs.org/en/).
+2. Install a Git client for your OS; see the [Git homepage](https://git-scm.com/downloads).
+3. Install dependencies using `yarn install`, as superuser. (Use `Open command prompt (CMD) (admin)` on Windows.) See the [Yarn homepage](https://yarnpkg.com/).
+4. Enable the corepack utility by running `corepack enable`, as superuser. See the [Corepack homepage](https://github.com/nodejs/corepack).
+
+To start working on this project, perform the following steps:
+1. Create a working directory for the project and change into it.
+2. Clone the project in the new directory `git clone https://github.com/eisopux/checker-framework-vscode.git` (adapt the URL to your specific fork and branch). 
+3. Change into the cloned directory `cd checker-framework-vscode`.
+4. Run `yarn global add vsce`.
```

Comment:
Does this step need to be performed in the project directory? Or should it be moved to the `set up your development environment` above? Can you find a link that explains what this does?